extensions/DynamicPageList/DPLMain.php

```diff
@@ -2311,20 +2275,26 @@ public static function dynamicPageList( $input, $params, $parser, &$bReset, $cal
         }
 
         // SELECT ... FROM
-        if ($acceptOpenReferences)
-                 // SELECT ... FROM
-        	if (count($aImageContainer)>0) {
-	        	$sSqlSelectFrom = "SELECT $sSqlCalcFoundRows $sSqlDistinct " . $sSqlCl_to . 'ic.il_to, ' . $sSqlSelPage . "ic.il_to AS sortkey" . ' FROM ' . $sImageLinksTable . ' AS ic';
-        	}
-        	else {
-	        	$sSqlSelectFrom = "SELECT $sSqlCalcFoundRows $sSqlDistinct " . $sSqlCl_to . 'pl_namespace, pl_title' . $sSqlSelPage . $sSqlSortkey . ' FROM ' . $sPageLinksTable;
-        	}
-        else
+        if ($acceptOpenReferences) {
+			// SELECT ... FROM
+			if ( count( $aImageContainer ) > 0 ) {
+				$sSqlSelectFrom =
+					"SELECT $sSqlCalcFoundRows $sSqlDistinct " . $sSqlCl_to . 'ic.il_to, ' .
```

Comment:
Is mixing single and double quotes side by side wanted here?

Reply:
I'd leave it as is for now

Reply:
Okay. No offense meant.